BranchInfo.props

```diff
@@ -12,7 +12,7 @@
     <IncludeBuildNumberInPackageVersion Condition="'$(StabilizePackageVersion)' != 'true'">true</IncludeBuildNumberInPackageVersion>
 
     <ReleaseSuffix>$(PreReleaseLabel)</ReleaseSuffix>
-    <ReleaseBrandSuffix Condition="'$(IncludePreReleaseLabelInPackageVersion)' == 'true'">Release Candidate 1</ReleaseBrandSuffix>
+    <ReleaseBrandSuffix Condition="'$(IncludePreReleaseLabelInPackageVersion)' == 'true'">Preview</ReleaseBrandSuffix>
```

Comment:
@mmitche Should this be Preview 2 to match the prerelease label?

Reply:
Yeah.